### PR TITLE
Skatepark: updated font sizes

### LINKED ISSUE #5414: Skatepark: Regression on font sizes
Looks like some change to font sizes on Blockbase has affected Skatepark:

Figma | Site
--- | ---
Screenshot 2022-01-31 at 12 53 39 | Screenshot 2022-01-31 at 12 53 03

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

This PR adds the changes done in https://github.com/Automattic/themes/pull/5300 to Skatepark. To test it check that the font sizes fit what's on the figma designs.

Screenshot 2022-01-31 at 16 08 16

#### Related issue(s):

Closes https://github.com/Automattic/themes/issues/5414

### CODE REVIEW

File: skatepark/theme.json [hunk 1]
```diff
@@ -232,6 +231,11 @@
 					"size": "16px",
 					"slug": "small"
 				},
+				{
+					"name": "Normal",
```
Comment (scruffian) [inline skatepark/theme.json]: We moved Normal out of the theme.json since this isn't a size that is supported by Gutenberg anymore. It should be in custom instead.

Comment (MaggieCabrera) [reply skatepark/theme.json]: ugh I read that backwards on the original PR...

File: skatepark/block-templates/search.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
 	<!-- wp:query {"query":{"perPage":10,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true}} -->
 	<!-- wp:post-template -->
 
-		<!-- wp:post-title {"level":3,"isLink":true,"style":{"typography":{"fontStyle":"normal","fontWeight":"800"}},"fontSize":"huge","style":{"spacing":{"margin":{"top":"1em"}}}} /-->
+		<!-- wp:post-title {"level":3,"isLink":true,"style":{"typography":{"fontStyle":"var(--wp--custom--font-sizes--normal)","fontWeight":"800"}},"fontSize":"x-large","style":{"spacing":{"margin":{"top":"1em"}}}} /-->
```
Comment (scruffian) [inline skatepark/block-templates/search.html]: I think this is a dodgy find and replace

Comment (MaggieCabrera) [reply skatepark/block-templates/search.html]: yeah something like that :/

File: skatepark/block-templates/search.html [hunk 3]
```diff
@@ -12,7 +12,7 @@
 	<!-- wp:query {"query":{"perPage":10,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true}} -->
 	<!-- wp:post-template -->
 
-		<!-- wp:post-title {"level":3,"isLink":true,"style":{"typography":{"fontStyle":"normal","fontWeight":"800"}},"fontSize":"huge","style":{"spacing":{"margin":{"top":"1em"}}}} /-->
+		<!-- wp:post-title {"level":3,"isLink":true,"style":{"typography":{"fontStyle":"normal","fontWeight":"800"},"fontSize":"var(--wp--custom--font-sizes--normal)","spacing":{"margin":{"top":"1em"}}}} /-->
```
Comment (scruffian) [inline skatepark/block-templates/search.html]: Shouldn't this be `x-large`?

File: blockbase/inc/patterns/footer-columns.php [hunk 4]
```diff
@@ -15,7 +15,7 @@
 		<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"60px","bottom":"60px"}}},"className":"pre-footer"} -->
 		<div class="wp-block-group alignwide pre-footer" style="padding-top:60px;padding-bottom:60px"><!-- wp:columns -->
 		<div class="wp-block-columns"><!-- wp:column -->
-		<div class="wp-block-column"><!-- wp:site-title {"style":{"typography":{"textTransform":"uppercase"}},"fontSize":"large"} /-->
+		<div class="wp-block-column"><!-- wp:site-title {"style":{"typography":{"textTransform":"uppercase"}},"fontSize":"medium"} /-->
```
Comment (scruffian) [inline blockbase/inc/patterns/footer-columns.php:18]: Is this change deliberate?

Comment (MaggieCabrera) [reply blockbase/inc/patterns/footer-columns.php:18]: yes, it was wrong compared to the design

File: skatepark/block-templates/search.html (hunk 3 above)
Comment (MaggieCabrera) [reply skatepark/block-templates/search.html]: mmh, now that you mention it, it should be just x-large, it was in fact me searching fontStyle: normal instead of fontSize: normal in this case, plus the markup for the whole block was off, so I'll undo that

**Review by scruffian — APPROVED** [review]
LGTM
